### PR TITLE
Feature: enable touch events for waveform container and pointer and segment markers on mobile devices

### PR DESCRIPTION
**Pull request by rfrei**
note:
- mouse events on mobile devices with connected USB mouse will only work with these Konva patches: https://github.com/rfrei/konva/commit/5bbb42d8fbca9c9d2e4e25c3b2f2323a6ca4cd10

### CODE REVIEW

File: src/main/views/helpers/mousedraghandler.js [hunk 1]
```diff
@@ -64,7 +65,12 @@ define(function() {
       return;
     }
 
-    this._mouseDownClientX = event.evt.clientX;
+    if (event.type === 'touchstart') {
+      this._mouseDownClientX = parseInt(event.evt.touches[0].clientX, 10);
```
Comment (chrisn) [inline src/main/views/helpers/mousedraghandler.js]: As `clientX` is already a number, I suggest using `Math.floor()` rather than `parseInt()`.

Comment (rfrei) [reply src/main/views/helpers/mousedraghandler.js]: Ok, thank you. I changed it.

**Comment by chrisn** [comment]
Many thanks for contributing!

**Comment by rfrei** [comment]
Thank you to for this nice software!
I have discovered another minor problem and will mention it in another issue: konvas _touchstart event handler does not call preventDefault() and because of that we get a second mousedown event after the touchstart event. This leads to short repeats of audio while playing and touching on the waveform. I come back soon.

**Pull request merged**